documentation/document.css

```diff
@@ -179,4 +179,28 @@
     
     .icon {
         color: black
+    }
+    
+    @media only screen and (max-width:930px) {
+        html {
+            font-size: 50%;
+        }
+    }
+    
+    @media only screen and (max-width: 768px) {
```

Comment:
Add the responsiveness in index page too, rest look good

Reply:
Yes, will do and raise new PR